src/lib/server/api/discord.ts

```diff
@@ -97,9 +131,20 @@ export async function logPendingConfessionViaHttp(
     authorId: Snowflake,
     label: string,
     description: string,
+    attachment: Attachment | null,
     botToken = DISCORD_BOT_TOKEN,
 ) {
     const customId = internalId.toString();
+    const fields = [
+        {
+            name: 'Authored by',
+            value: `||<@${authorId}>||`,
+            inline: true,
+        },
+    ];
+    if (attachment) {
```

Comment:
Similar concern regarding explicit conditions.
```suggestion
    if (attachment !== null) {
```